modules/ROOT/pages/consuming-a-soap-api.adoc

```diff
@@ -0,0 +1,17 @@
+= Consuming a SOAP API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint, studio, connectors, soap, wsdl, api
+
+If you want to consume an API that does not already have its own Anypoint Connector, use the *Web Service Consumer* connector to consume any SOAP API from within your Mule application.
+
+Using the information contained in the API's WSDL, this connector enables you to configure a few details to establish the connection you need for consuming a service from within your Mule application. +
+First, identify the location of the Web service's WSDL file, then ask the Web Service Consumer to configure itself from the WSDL – host, port, address.
```

Comment:
"ask" would normally be "make a request" or something similar, as "request" is more typical in this context

Reply:
This needs to be more clear: how does your audience "ask" the WSC to configure itself? Is there a command?
See suggested replacement for all of this above.